Clinical-Genomics/scout
# return always prio cases

This is an alternative way to fix #1494. 
I didn't like much the idea of adding an extra query to the whole process so I wrote this attempt to do the same without this step. I don't know if it works, should be tested on stage!

**How to test**:
1. Go on scout stage and make sure that 1 old case is prioritized for cust000 : 15026-miptest
1. From master branch. If you limit the case query to 1 case: (https://scout-stage.scilifelab.se/cust000?query=&limit=1) then the case returned is the latest edited case (15078-mip7), not the prio one.
1. Install this branch and run the same query. The prio sample should be returned.
1. Bonus test. Run this query: https://scout-stage.scilifelab.se/cust000?query=&limit=2 and 2 samples will be returned: both the prio and the latest edited (15078-mip7)

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
